### PR TITLE
fill empty listings page if no listings available

### PR DESCRIPTION
**Pull request by Jorgro**
## Proposed Changes
- Fill the empty Listings (Verv) page if no listings available

## Types of Changes

- [x] New feature
- [ ] Bug fix
- [ ] Enhancement/optimization
- [ ] Refactor
- [ ] Style
- [ ] Build/dependencies
- [ ] Other

## Issue(s) fixed or closed by this PR
-

## Checklist

- [x] I have added tests to cover my changes (for features/bugs)
- [x] I am pleased with the readability of the code (if not, state where you need input)
- [x] I have tested the changes and verified that they work and do not break anything (to the best of my ability)

### CODE REVIEW

File: frontend/src/components/pages/listings/index/Listings.tsx [hunk 1]
```diff
@@ -54,11 +56,24 @@ const Listings: React.FC<{
...
         ))}
+      {data && data.listings.length == 0 && (
+        <>
+          <Grid item>
+            <Typography variant="body1" align="center">
+              Det er for øyeblikket ingen verv tilgjengelige.
+            </Typography>
+          </Grid>
+          <Grid item md={6} xs={10}>
+            <Image src={_EmptyStreet} alt="Empty street" />
```
Comment (larwaa) [inline frontend/src/components/pages/listings/index/Listings.tsx:73]: Should be blank for decorative images
```suggestion
            <Image src={_EmptyStreet} alt="" />
```

File: frontend/src/components/pages/listings/index/Listings.tsx [hunk 2]
```diff
@@ -54,11 +56,24 @@ const Listings: React.FC<{
         </Grid>
       )}
       {data &&
+        data.listings.length > 0 &&
```
Comment (larwaa) [inline frontend/src/components/pages/listings/index/Listings.tsx:59]:
```suggestion
      {data?.listings.length > 0 &&
```

**Review by larwaa — CHANGES_REQUESTED** [review]

File: frontend/src/components/pages/listings/index/Listings.tsx [hunk 3]
```diff
@@ -4,6 +4,8 @@ import { Listing } from "@interfaces/listings";
 import { useQuery } from "@apollo/client";
 import { Grid, Typography, makeStyles, CircularProgress, Button } from "@material-ui/core";
 import Link from "next/link";
+import Image from "next/image";
+import _EmptyStreet from "public/illustrations/EmptyStreet.svg";
```
Comment (larwaa) [inline frontend/src/components/pages/listings/index/Listings.tsx:8]: Only need the underscore for stuff that doesn't start with a letter
```suggestion
import EmptyStreet from "public/illustrations/EmptyStreet.svg";
```

File: frontend/src/components/pages/listings/index/Listings.tsx [hunk 4]
```diff
@@ -53,12 +55,24 @@ const Listings: React.FC<{
...
         </Grid>
       )}
-      {data &&
+      {data?.listings.length > 0 &&
         data.listings.map((listing) => (
           <Grid container item key={listing.id} md={5} sm={7} xs={10}>
             <ListingItem listing={listing} />
           </Grid>
         ))}
+      {data && data.listings.length == 0 && (
```
Comment (Jorgro) [inline frontend/src/components/pages/listings/index/Listings.tsx:64]:
```suggestion
      {data?.listings.length == 0 && (
```

**Review by Jorgro — COMMENTED** [review]
Great work!

**Review by larwaa — APPROVED** [review]
Impressive front end work, LGTM!

**Pull request merged**